File: packages/cli/test/fixtures/ranking-corpora/python/rankingpy/handlers/delete.py
"""Removes a user by id."""

from rankingpy.util.logger import log_error, log_info


def delete_user(user_id: int) -> None:
    if user_id <= 0:
        log_error("delete_user", ValueError("id must be positive"))
        return
    log_info("deleted user")

File: packages/cli/test/fixtures/ranking-corpora/rust/src/delete.rs
use crate::util;

/// Removes a user by id, logging the outcome through the shared `util`
/// module.
pub fn delete_user(id: u32) -> Result<(), String> {
    if id == 0 {
        let err = "id must be positive".to_string();
        util::log_error("delete_user", &err);
        return Err(err);
    }
    util::log_info("deleted user");
    Ok(())
}
